packages/horizon/contracts/interfaces/ITAPCollector.sol

```diff
@@ -34,6 +43,36 @@ interface ITAPCollector is IPaymentsCollector {
         bytes signature;
     }
 
+    /**
+     * @notice Emitted when a signer is authorized to sign RAVs for a payer
+     * @param authorizedSigner The address of the authorized signer
+     * @param payer The address of the payer authorizing the signer
+     */
+    event AuthorizeSigner(address indexed authorizedSigner, address indexed payer);
```

Comment:
nit: I see you've used the same event signatures as in TAP verifier, but I wonder if we could use the opportunity to make them consistent with other events in two ways:
1) Ensure order is consistent, e.g. always have payer, authorizedSigner, thawEndTimestamp in that order
2) Use more descriptive verb tenses, e.g. SignerAuthorized, SignerThawing, SignerThawingCanceled, SignerRevoked